app/views/works/index.html.erb

```diff
@@ -0,0 +1,73 @@
+<h3>List of Works</h3><br>
+
+<h4>Albums</h4>
+<table>
+  <thead>
+    <tr>
+      <th width="200">Votes</th>
+      <th>Title</th>
+      <th width="150">Created By</th>
+      <th width="150">Published</th>
+      <th width="150">Upvote</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @albums.each do |album|%>
+    <tr>
+        <td><%= album.vote_count %></td>
+        <td><%= link_to album.title, work_path(album.id) %></td>
+        <td><%= album.creator %></td>
+        <td><%= album.publication_year %></td>
+        <td><%= link_to("Upvote", work_votes_path(album.id), class: "button", method: :post) %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<h4>Books</h4>
+<table>
+  <thead>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?